backend/prompt_studio/prompt_studio_core/prompt_studio_helper.py

```diff
@@ -768,6 +782,7 @@ def _fetch_single_pass_response(
         answer = responder.single_pass_extraction(payload)
         # TODO: Make use of dataclasses
         if answer["status"] == "ERROR":
-            raise AnswerFetchError()
+            error_message = answer.get("error", "")
```

Comment:
It'll be good to default some error message - I'm not sure how an empty error message would be taken up.
This default could be specific to this use case like `Failed to perform single pass extraction` - doesn't help much but it'll not be `Something went wrong` atleast

Reply:
@chandrasekharan-zipstack  If empty it will use default error automatically.